### ISSUE TITLE
Search not working

### ISSUE THREAD

**Issue by EatMoreSugarcane** [Ashod/Now-Reading-Redux#3]
When I try to add a new book and search using ISBN I get the following debug error message:

<pre>
Amazon Search XML:
string(18) "Amazon Search XML:"
string(524) "<?xml version="1.0"?>

<ItemSearchErrorResponse xmlns="http://ecs.amazonaws.com/doc/2009-03-31/">
<Error>
<Code>
SignatureDoesNotMatch</Code>
<Message>
The request signature we calculated does not match the signature you provided. Check your AWS Secret Access Key and signing method. Consult the service documentation for details.</Message>
</Error>
<RequestID>
6ed5c0b2-b17a-4da7-ae45-9b2093883647</RequestID>
</ItemSearchErrorResponse>
"</pre>

I have checked that my AWS access key ID and secret access key are correct.  I created the account today.

I am using WordPress 3.0.5.

**Comment by Ashod** [comment]
Hi,

Amazon clearly doesn't like the ID and Key it's getting. Since there has been no release with a broken search, I suspect there is something wrong with the values you're using.

There are two possibilities that I can think of. First, the case may be that Amazon hasn't activated your ID/Key yet. For that you'll need to check the documentation if there is any info on activation delay. The second, which is more likely, is that you swapped the two values, or left a character or two out. The Access Key ID is the shorter one of the two.

My only advice at this point is to go back to the registration pages (links next to the fields in the options page) and read carefully. Then copy paste and compare the values as carefully as you can.

For all I know the search is working and was never broken. Please don't hesitate to get back to me if the issue isn't resolved.

**Comment by EatMoreSugarcane** [comment]
Still not working.  It has been a week, so it can't be an activation delay.  I tried creating new keys and trying those as I noticed that my secret key had a '/' in it and I thought that might be a problem, but that didn't make a difference either.  If it's alright, I'd like to send you my key pair to try to confirm that it's a problem with Amazon and not my system.  My email address is chris@northfolk.ca.  If you contact me there I will send you the keys.
